build/packages-template/bbb-config/include_default.nginx

```diff
@@ -0,0 +1,9 @@
+# The purpose of this file is to be included in an nginx site configuration.
+# you may copy this file to /etc/bigbluebutton/nginx/include_default.nginx and
+# adapt it to your needs
+#
+# If you are running a cluster setup, you need to change bbb-html5.nginx
+# copy it to /etc/bigbluebutting/nginx/bbb-html5.nginx and include all the
```

Comment:
AFAIK, often times, we use clusters, wouldn't it be nicer to not have this other step? You could just do that in the after-install part, maybe? (If it doesn't break anything, of course) What do you think?

Reply:
this refers to https://docs.bigbluebutton.org/admin/clusterproxy.html where different location schemes have to be used. This is not related to scalelite or any other loadbalancer